generators/generator-base.js

```diff
@@ -1683,6 +1684,24 @@ module.exports = class extends PrivateBase {
     }
 
     /**
+     * Print an info message.
+     *
+     * @param {string} value - message to print
+     */
+    info(msg) {
```

Comment:
Yes I thought about it too but I did not want to impact too much code. From the log statuses provided by yeoman, I like `ok()` and `info()` and `maybe error()` (I don't remember it well) but I'm not sure the others are that useful, for instance I tried to use `invoke()` for all git calls but finally preferred `info()`. 

Also I must admit that I did not understand how their indentation system works with [emitted events](https://github.com/yeoman/environment/blob/master/lib/util/log.js#L51-L52).